src/libs/ReportUtils.ts

```diff
@@ -9156,13 +9201,21 @@ function getIOUReportActionDisplayMessage(reportAction: OnyxEntry<ReportAction>,
         const {amount, currency} = originalMessage?.IOUDetails ?? originalMessage ?? {};
         const formattedAmount = convertToDisplayString(Math.abs(amount), currency) ?? '';
 
+        // This will be fixed as part of https://github.com/Expensify/Expensify/issues/507850
+        // eslint-disable-next-line deprecation/deprecation
```

Comment:
Please do not add more usages of `getPolicy()`. It has been deprecated for a reason and adding more usages is just increasing the work that other people need to do.